subprojects/docs/src/docs/userguide/troubleshootingDependencyResolution.adoc

```diff
@@ -15,83 +15,93 @@
+Managing dependencies in a project can be challenging. This chapter describes techniques for troubleshooting issues you might encounter in your project as well as best practices for avoiding common problems.
 
 [[sub:version_conflicts]]
 === Resolving version conflicts
 
-Conflicting versions of the same jar should be detected and either resolved or cause an exception. If you don't use transitive dependency management, version conflicts are undetected and the often accidental order of the classpath will determine what version of a dependency will win. On a large project with many developers changing dependencies, successful builds will be few and far between as the order of dependencies may directly affect whether a build succeeds or fails (or whether a bug appears or disappears in production).
+Conflicting versions of the same module should be detected and either resolved or cause an exception. If you don't use transitive dependency management, version conflicts are undetected and may influence the order of the classpath determining what version of a dependency will be selected. In short, version conflicts can cause classpath issues and are extremely hard to debug.
 
-If you haven't had to deal with the curse of conflicting versions of jars on a classpath, here is a small anecdote of the fun that awaits you. In a large project with 30 submodules, adding a dependency to a subproject changed the order of a classpath, swapping Spring 2.5 for an older 2.4 version. While the build continued to work, developers were starting to notice all sorts of surprising (and surprisingly awful) bugs in production. Worse yet, this unintentional downgrade of Spring introduced several security vulnerabilities into the system, which now required a full security audit throughout the organization.
-
-In short, version conflicts are bad, and you should manage your transitive dependencies to avoid them. You might also want to learn where conflicting versions are used and consolidate on a particular version of a dependency across your organization. With a good conflict reporting tool like Gradle, that information can be used to communicate with the entire organization and standardize on a single version. _If you think version conflicts don't happen to you, think again._ It is very common for different first-level dependencies to rely on a range of different overlapping versions for other dependencies, and the JVM doesn't yet offer an easy way to have different versions of the same jar in the classpath (see <<sub:dependency_management_and_java>>).
-
-Gradle offers the following conflict resolution strategies:
+Gradle can <<managing_transitive_dependencies,manage transitive dependencies>> for you with the help of the following conflict resolution strategies:
 
 * _Newest_: The newest version of the dependency is used. This is Gradle's default strategy, and is often an appropriate choice as long as versions are backwards-compatible.
```

Comment:
Relating to my comment above: I would either mark this as the default or remove this list of two items. And just have it in the text that _you can change the default to "fail"_.

Reply:
done